src/examples/java/io/nats/examples/autobench/NatsAutoBench.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) {
                                                     connectionTimeout(Duration.ofSeconds(1)).
                                                     noReconnect();
 
-            /**
+            /*
```

Comment:
If this was around an api javadoc and maybe broke the doc, then absolutely, but again, Non client production code, only if already changing or fixing the class.